modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -197,6 +197,65 @@ definition {
 		}
 	}
 
+	@description = "LPS-173537 - Verify creating a custom object entry triggers an action to update an Account entry"
+	@priority = 5
+	test CanAddAccountEntryAfterUpdatingCustomObjectEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a Custom Object and an object entry") {
```

Comment:
`custom object`

We don't need to capitalize `Custom Object` because it isn't a proper noun.